mitmproxy/contentviews/base.py

```diff
@@ -35,6 +35,35 @@ def __call__(self, data: bytes, **metadata) -> TViewResult:
         raise NotImplementedError()  # pragma: no cover
 
 
+def format_pair(item: TViewLine) -> typing.Iterator[TViewLine]:
```

Comment:
This method should be `format_pairs` and not `format_pair`, as it formats multiple (key, value) pairs.

Reply:
Second, `item` is a bad name for a list. `items` would work, or `pairs`, or something like that.

Reply:
Also, the TViewLine type is used to indicate `(style, text)` tuples for rendering. That's not really what we have as input here, which should just be an `Iterable[Tuple[TTextType,TTextType]]`.

Reply:
okay!

Reply:
Sorry i misinterpreted TViewLine type.